src/core/event.cpp

```diff
@@ -325,8 +325,7 @@ void on_parameter_change(int field) {
     break;
   case FIELD_NODEGRID:
     grid_changed_n_nodes();
-    cells_on_geometry_change(CELL_FLAG_GRIDCHANGED);
-    break;
+    cells_re_init(CELL_STRUCTURE_CURRENT, cell_structure.min_range);
   case FIELD_MINNUMCELLS:
   case FIELD_MAXNUMCELLS:
     cells_re_init(CELL_STRUCTURE_CURRENT, cell_structure.min_range);
```

Comment:
is this fallthrough intentional? `cells_re_init()` is now called twice with the same arguments if case `FIELD_NODEGRID` matches. Maybe you meant to delete L328?

Reply:
Ah yes, good catch.